### PR TITLE
improve mountains ⛰️

### PR DESCRIPTION
**Pull request by Makio64**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -81,32 +81,32 @@ const perspective = Math.floor(FXRandomBetween(0.01, 0.08) * 100) / 100
...
-  Makio64Style,
-  MandyBrigwellStyle,
-  BridgeTunnelStyle,
-  DreyStyle,
-  LunareanStyle,
-  GrosggStyle,
-  bfosStyle,
-  DevnullStyle,
-  Nobody,
+  // AdaAdaAdaStyle,
```
Comment (camilleroux) [inline src/index.js:84]: You can't commit these changes

**Review by camilleroux — CHANGES_REQUESTED** [review]
do not commit the changes to index.js

File: src/index.js (hunk 1 above)
Comment (Makio64) [reply src/index.js]: Ooops, I just reverted the change on index :)

thanks!

File: src/index.js [hunk 2]
```diff
@@ -98,7 +98,7 @@ const stylesClasses = [
   BoilerplateStyle,
   DemoStyle,
   AzeemStyle,
-  Makio64Style,
+  Makio64Style
```
Comment (camilleroux) [inline src/index.js:101]: comma is missing I guess

**Review by camilleroux — CHANGES_REQUESTED** [review]

**Pull request merged**